src/secondAssignment_14March2023/FindDestinationCity.java

```diff
@@ -0,0 +1,22 @@
+package secondAssignment_14March2023;
+
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+public class FindDestinationCity {
+
+	public static void main(String[] args) {
```

Comment:
remove .class file and work on the coding guideline points

Reply:
Done